frontend/src/components/pipelines-or-deployments/pipelines/Pipelines.jsx

```diff
@@ -103,6 +100,28 @@ function Pipelines({ type }) {
       });
   };
 
+  const handleStatusRefresh = (pipelineId) => {
+    const fieldsToUpdate = {
+      last_run_status: "processing",
+    };
+    handleLoaderInTableData(fieldsToUpdate, pipelineId);
+
+    getPipelineData(pipelineId)
+      .then((res) => {
+        const data = res?.data;
+        fieldsToUpdate["last_run_status"] = data?.last_run_status;
+        fieldsToUpdate["last_run_time"] = data?.last_run_time;
+      })
+      .catch((err) => {
```

Comment:
@chandrasekharan-zipstack We need to updated the `last_run_status` in case of failure. If not, the loader will keep running when failed. Please refer [handleSync](https://github.com/Zipstack/unstract/blob/234af37e83ae994ff6b8c2fc743a6b1aa0200a02/frontend/src/components/pipelines-or-deployments/pipelines/Pipelines.jsx#L95) function.

Reply:
This is done.